ckan/requirements.txt

```diff
@@ -93,7 +94,7 @@ Routes==1.13
 rq==1.0
 s3transfer==0.6.0
 setuptools==65.6.3
-Shapely==1.8.5.post1
+shapely==2.0.0
```

Comment:
This seems potentially dangerous, but tests passed so 🤷‍♂️

Reply:
I would think so too, but hey, +1 to the snyk script, right? 😆